docs/source/how_to_guides/optimization/how_to_use_constraints.ipynb

```diff
@@ -1,29 +1,28 @@
 {
  "cells": [
   {
-   "cell_type": "code",
-   "execution_count": 1,
+   "cell_type": "markdown",
```

Comment:
Can you add one section at the beginning that shows how maximize or minimize is called with a list of constraints. several people have told me that that does not become clear from the tutorial. 

Also, I think the notebook is too long. What do you think about switching from a notebook to a tab presentation like [here](https://sphinx-tabs.readthedocs.io/en/latest/)? Maybe this would be a good idea in several places (e.g. to switch between different optimization examples). And maybe there is a way to make the design nicer? It's just the first tool I found ...